packages/web-components/stencil.config.ts

```diff
@@ -9,7 +9,10 @@ export const config: Config = {
   globalStyle: 'src/global/main.css',
   plugins: [
     sass({ 
-      includePaths: ["../../node_modules/@uswds/uswds/packages/"],
+      includePaths: [
+        '../../node_modules/@uswds/uswds/packages/',
+        'src/global/'
```

Comment:
Have we explored including the `css-library` as a dependency similar to the way we are consuming the `uswds` packages here? 

So instead of `src/global` it would just import the css library package? '../../node_modules/@department-of-veterans-affairs/css-library'. 

https://www.npmjs.com/package/@department-of-veterans-affairs/css-library

Reply:
Yes, I did it in this test [PR](https://github.com/department-of-veterans-affairs/component-library/pull/581/files)

```
includePaths: [
        '../../node_modules/@uswds/uswds/packages/',
        '../../node_modules/@department-of-veterans-affairs/css-library/src/tokens/',
      ],
```

Reply:
We agreed to create the file locally in `web-components` first and then it we like it, then we will pull the file directly from `css-library`

Reply:
Understood. Is there a ticket already about publishing the `css-library` npm package yet? We might want to add a comment there to follow up on updating the way this is imported to save us the trouble of having to manually generate the variables stylesheet. I think doing that might become tedious at some point especially if we are needing to add additional colors to the library (like the error color).